client/vscode-lib/src/util/esmToCommonJS.test.ts

```diff
@@ -14,12 +14,19 @@ describe('esmToCommonJS', () => {
         expect(esmToCommonJS(esm)).toBe(expected)
     })
 
+    test("aliased import statements", () => {
```

Comment:
CI is failing due to needing to run `pnpm biome`. I'll attempt the fixes as suggestions.
```suggestion
    test('aliased import statements', () => {
```